vignettes/safestats-vignette.Rmd

```diff
@@ -188,30 +186,30 @@ Due to sampling error, the average number of times that $S > 1 / \alpha$ might n
 
 ## <a name = "optStopT">Safe Tests Allow for Optional Stopping without Inflating the Type I Error Rate above the Tolerable $\alpha$-Level</a>
 <!--### Data under the alternative: <a name = "optStopT">Stopping the experiment early</a>-->
-What makes safe tests particularly interesting is that they allow for early stopping without the test exceeding the tolerable type I error rate of $\alpha$. This means that the evidence can be monitored as the data comes in, and when there is a sufficient amount of evidence against the null, thus, $S > 1/\alpha$, the experiment can be stopped early, which therefore increases efficiency. We highlight this with our design object 
+What makes the safe tests in this package particularly interesting is that they allow for early stopping without the test exceeding the tolerable type I error rate of $\alpha$. This means that the evidence can be monitored as the data comes in, and when there is a sufficient amount of evidence against the null, thus, $S > 1/\alpha$, the experiment can be stopped early, which therefore increases efficiency.
+
+> Note that <a name = "martingale">not all s-values necessarily allow for optional stopping</a>: this only holds for some special s-values, that are also _test martingales_. More information can be found, for example, in the first author's [master thesis](https://www.universiteitleiden.nl/binaries/content/assets/science/mi/scripties/statscience/2019-2020/thesis_rjturner_for_publication.pdf), Chapter 5.
+
+We highlight this with our design object 
 
 ```{r}
 designObj
 ```
 
 ### Safe tests detect the effect early if it is present: deltaTrue equal to deltaMin
-<!--#### Optional stopping: True effect size equals minimal clinical relevant effect size-->
+<!--#### Optional stopping: True effect size equals minimal clinically relevant effect size-->
 The following code replicates 1,000 experiments and each data set is generated with a true effect size that equals the minimal clinical-relevant effect size of $\delta_{\min}=9/(15 \sqrt{2}) = 0.42$. The safe test is applied to each data set sequentially and if the s-value is larger than $1 / \alpha$, the experiment is stopped. If the s-value does not exceed $1 / \alpha$, the experiment is run until all samples are collected as planned.
 
 ```{r}
-# alpha <- 0.05
```

Comment:
The alpha, beta and deltaMin are defined above, but I wanted to keep them in the code, so people know what their value is. Perhaps my intention wasn't clear and we should add something such as

    # Recall:
    # alpha <- 0.05
    # beta <- 0.2
    # deltaMin <- 9/(sqrt(2)*15)      # = 0.42